analyzer/src/funTest/kotlin/managers/Extensions.kt

```diff
@@ -25,6 +25,9 @@ import java.io.File
 
 import org.ossreviewtoolkit.analyzer.PackageManager
 import org.ossreviewtoolkit.model.ProjectAnalyzerResult
+import org.ossreviewtoolkit.model.yamlMapper
+
+fun Any?.toYaml() = yamlMapper.writeValueAsString(this)!!
```

Comment:
How about giving this a bit more distinctive name to avoid name clashes? (As any is quite broad)

Reply:
But there are no name clashes, and I want to have a short name.

Reply:
To be frank (no pun intended), I'd vote for simply dropping this commit. Also see my [comment below](https://github.com/oss-review-toolkit/ort/pull/2711#discussion_r438642902).